### DISCUSSION TITLE
skywalking-client-js cors error

### DISCUSSION THREAD

**Discussion by gamlungJoeng** [apache/skywalking#10097] [General]
1. I get a APM( it's a tar.gz file, have some jar libs ) and update application.yml next start it. 
2. I generate a react-app, Add skywalking-client-js lib and register.
3. Now, I open devTools and see network have CORS error. 
How should I modify the response header to fix? Need your help very much

**Comment by gamlungJoeng** [comment]
![image](https://user-images.githubusercontent.com/118876071/205611853-a10bf5fa-ecb4-4314-ab34-1dcf19252bd4.png)
log just so

**Comment by gamlungJoeng** [comment]
java.util.concurrent.CompletionException: com.linecorp.armeria.client.UnprocessedRequestException: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:12800

I get a error

**Comment by mayur-gupta-smartsense** [comment]
I am having the same issue, please help: `Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: localhost/127.0.0.1:12800`

**Reply by gamlungJoeng** [reply]
My reason is that the browser's cross-domain restrictions prevent the request from being sent. Fixing it is very simple, you can use nginx as a proxy to complete request forwarding.
If your Agent is not skywalking-client-js, you can try to record the log when the request is sent, and open the logs directory under the root directory of skywalking's OAP service, and check the log file inside to see if you have received a request record from the Agent
